src/components/callout/callout.e2e.ts

```diff
@@ -0,0 +1,77 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('limel-callout', () => {
+    it('renders a callout with the default "Info" heading and no description', async () => {
+        const page = await newE2EPage({ timeout: 10000 });
```

Comment:
I added unit tests with help of Chat GPT. Perhaps I'm officially the first one at Lime who does this? 😅 
Anyhow, its suggestion was adding `timeout: 10000` to all tests. Is it too long? Does it make `npm run test` run slowly? Is there a better way? You tell me. I'm not a good test writer.

Actually, don't tell me. Just fix it up please. I don't care. ;)

Reply:
> I added unit tests with help of Chat GPT. Perhaps I'm officially the first one at Lime who does this? 😅

way to go!

> Is it too long?

yeah, 10 secs seems excessive. I'm not familiar with the e2e tests yet, so I might leave that for someone else 😊

Reply:
This was what the AI said: 👇 😄  (First, there was not timeout.)
<img width="834" alt="image" src="https://user-images.githubusercontent.com/35954987/228044956-e0f7382a-f631-424c-b7bb-62ae4acc59db.png">

Reply:
I don't think Chat GPT is good enough to help beginners to write code like that. It still makes too many mistakes. Fun to try though, and it certainly won't be long before it performs _much_ better…

Copilot X for GitHub was revealed recently, and from the little I saw in a video, it seems very competent, and since it lives in VS Code (or other IDE), it automatically has basically the whole repo as "context" for the suggestions it makes.

Reply:
For now though, I think you're better off looking att our existing code and asking for help 😊

I know you asked us to just fix it. If it's still here when I get around to it, I will, but I have other stuff too.

Reply:
Removed the timeout and do it like the limel-button does